packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -13,19 +13,17 @@ Before you can sign a user in, you need to create an Okta app integration that r
-1. Add the **Base URI** of your application during local development, such as `http://localhost:8080`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+1. In the **Trusted Origins** section, add the **Base URI** of your application during local development, such as `http://localhost:8080`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`. These URIs are added as trusted origins in your Okta org and can be managed by navigating to **Security** > **API** and selecting the **Trusted Origins** tab. See [Enable Trusted Origins](#enable-trusted-origins).
+
+1. In the **Assignments** section, select **Allow everyone in your organization to access** for everyone to access your app, or **Limit access to selected groups** and specify the groups that can access your app.
 1. Click **Save** to finish creating the Okta app integration.
 1. On the **General** tab, scroll to **General Settings** and click **Edit**.
-1. In the **Allowed grant types**, make sure that **Authorization Code** and **Refresh Token** are selected.
-
-   This enables PKCE flow for your application and the ability to refresh the access token when it expires without prompting the user to reauthenticate. [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation) is automatically set as the default refresh token behavior.
-
-    Refresh token rotation for SPA is an <ApiLifecycle access="ea"/> feature.
-    You can enable an [Early Access (Self-Service)](/docs/reference/releases-at-okta/#early-access-ea) feature for your org in the **Settings** > **Features** page inside the Admin Console.
-
+1. If **Refresh Token** is selected as a **Grant type**, the **Refresh Token** section appears. [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation) is automatically set as the default refresh token behavior.
     > **Note:** The default number of seconds for the **Grace period for token rotation** is set to 30 seconds. You can [change the value](/docs/guides/refresh-tokens/refresh-token-rotation/#enable-refresh-token-rotation) to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there is no grace period.
+1. In the **LOGIN** section, specify an **Initiate login URI** to have Okta initiate the sign-in flow. When Okta redirects to this URI (for example, `https://example:0000.com/login`), the client is triggered to send an authorize request. This URI is also used when users reset their passwords while signing in to the app. Okta redirects the user back to this URI after the password is reset so that the user can continue to sign in.
```

Comment:
Just a quick question around this line:`In the **LOGIN** section, specify an **Initiate login URI** to have Okta initiate the sign-in flow`. Are we referring to the App Wizard Form here? I'm asking because we use a slightly different labeling if so.

Reply:
Yes, this is App Wizard, after you create the app. When you go back to edit the General Settings, I see this label in  Preview Org 2021.06.0.

Reply:
Should I add a step to set **Login initiated by** "Okta and App"? before the **Initiate login URI** can be set?

Reply:
oh ok I see so we're referring to the General Settings page then. Thanks for clarifying.

Reply:
> Should I add a step to set **Login initiated by** "Okta and App"? before the **Initiate login URI** can be set?

Sorry I missed this question before, just saw the email notification now. I think this is a good point, to my understanding setting the Initiate login URI only makes sense if login could be initiated by Okta or the App. @haishengwu-okta can help confirm this too .

Reply:
No worries, I can followup with @haishengwu-okta  and update the doc in another PR. Thanks